.gitignore

```diff
@@ -4,4 +4,10 @@ runinfo
 test/outputs
 .ropeproject/
 build/
-cwl/
+ceci.egg-info*
+dist
+log.txt
+test/log*
+.ipynb*
```

Comment:
I'm not sure we want ipynb to be in the gitignore - we may well want to include some of these with the project later.  Ditto the .txt files.

Reply:
yes, that's fair